.github/workflows/build-package.yml

```diff
@@ -52,7 +53,7 @@ jobs:
       - name: Clone and install tools branch
         run: |
           gcc --version
-          git clone -b v0.7.22 --depth 1 https://github.com/citusdata/tools.git ../tools
+          git clone -b add-pg15-scripts --depth 1 https://github.com/citusdata/tools.git ../tools
```

Comment:
Once https://github.com/citusdata/tools/pull/239 lands, we will create and push a tag on tools repo.

Reply:
Any news about that?

Reply:
I pinged @gurkanindibay who worked on CI readiness for PG15.

Reply:
I re-run the tests, and apparently we are missing some packages for zstd that breaks our builds on PG15 now.

https://github.com/citusdata/pg_auto_failover/actions/runs/3150979932

I do not have much bandwidth, but I guess @gurkanindibay will update this PR by:
a) updating the `Pre Install Steps` in this file to install missing packages
b) update the tools tag to latest available version.

Reply:
Do you think we still need the special branch `add-pg15-scripts` of the tools repository, rather than the current/latest tagged version of it?

Reply:
@hanefi  @DimCitus  Build is successful now

Reply:
Thanks so much for fixing it @gurkanindibay ; can you approve the Pull Request so that I can then merge it? thanks!

Reply:
I finally bypassed the rule for extra approval.